lib/network.go

```diff
@@ -2245,7 +2250,7 @@ type MsgBitCloutTxn struct {
 	// inputs to the transaction. The exception to this rule is that
 	// BLOCK_REWARD and CREATE_bitclout transactions do not require a signature
 	// since they have no inputs.
-	Signature *btcec.Signature
+	Signature []byte
```

Comment:
Huh. This seems like an interesting change. What prompted this?

Reply:
Ah, it was the custom signature format right? Given that we removed that I would revert this to how it was.

Reply:
Yup, exactly. Reverted.